pymc4/distributions/__init__.py

```diff
@@ -1 +1,8 @@
-from . import base
+__backend = 'tensorflow'
```

Comment:
We can leave it now as is or use Keras approach:
https://github.com/keras-team/keras/blob/master/keras/backend/load_backend.py#L75

Reply:
Added enviroment checking option. Although only tensorflow backend is supported for now.